main.go

```diff
@@ -72,6 +77,41 @@ func dataVisHandler(w http.ResponseWriter, req *http.Request) {
 }
 
 func apiHandler(w http.ResponseWriter, req *http.Request) {
+	log.DebugR(req, "returning api help text", log.Data{
+		"host": req.Host,
+		"path": req.URL.Path,
+	})
 	w.WriteHeader(410)
 	w.Write([]byte(apiResponse))
 }
+
+func visualAssetHandler(w http.ResponseWriter, req *http.Request) {
+	dest := "https://static.ons.gov.uk/visual/" + mux.Vars(req)["uri"]
```

Comment:
does this return the uri + any query string or just the uri

Reply:
just picks up the URI, query string parameters are completely ignored